site/content/en/docs/Guides/Client SDKs/rust.md

```diff
@@ -58,3 +53,37 @@ if sdk.shutdown().is_err() {
     println!("Could not run Shutdown");
 }
 ```
+
+To [set a Label]({{< relref "_index.md#setlabel-key-value" >}}) on the backing `GameServer` call `sdk.set_label(key, value)`.
```

Comment:
Small thing, this will need to be wrapped in a feature shortcode for "0.10.0" so it's hidden until the next release.

For example:
https://agones.dev/site/docs/contribute/#within-a-page